modules/installation/pages/install-server-unified.adoc

```diff
@@ -63,7 +61,7 @@ click [guimenu]``Software`` and select the [guimenu]``X Window System``
 pattern.
 ====
 +
```

Comment:
```suggestion
```
If the next sentence should become a step of it's own, we'd better remove the `+` connector.